docs/tools/benchmarking.md

```diff
@@ -0,0 +1,49 @@
+# tm-bench
+
+Tendermint blockchain benchmarking tool:
+
+- https://github.com/tendermint/tools/tree/master/tm-bench
+
+For example, the following:
+
+    tm-bench -T 10 -r 1000 localhost:26657
+
+will output:
+
+    Stats             Avg        Stdev      Max
+    Block latency     6.18ms     3.19ms     14ms
+    Blocks/sec        0.828      0.378      1
+    Txs/sec           963        493        1811
+
+## Quick Start
+
+[Install Tendermint](https://github.com/tendermint/tendermint#install)
+
+then run:
+
+    tendermint init
+    tendermint node --proxy_app=kvstore
+
+    tm-bench localhost:26657
+
+with the last command being in a seperate window.
+
+## Usage
+
+    tm-bench [-c 1] [-T 10] [-r 1000] [endpoints]
```

Comment:
outdated

Reply:
its the latest from tools: will update in #1921